util/common/src/test/resources/log4j2.properties

```diff
@@ -14,12 +14,21 @@ rootLogger.appenderRef.stdout.ref=STDOUT
 logger.jboss.name=org.jboss
 logger.jboss.level=info
 
+### Search logger for test utils: org.hibernate.search.util.impl.test.logging.Log
+# NOTE: we want to have a separate configuration for this particular logger,
+# since otherwise it will pick up the configs from `logger.search.name`
+# resulting in enabled trace logging in tests like HashTableUniformityTest.
+# And that will slow things down too much!
+logger.searchutil.name=org.hibernate.search.util.impl.test.logging
+# See the appender threshold above to actually log trace events
+logger.searchutil.level=info
+
 ### Search logger
 logger.search.name=org.hibernate.search
 # See the appender threshold above to actually log trace events
-logger.search.level=trace
+logger.search.level=debug
```

Comment:
The point of this was to allow us to enable tracing everywhere, just by changing the appender level.

If you want to remove that trick, fine, but then maybe remove the threshold filter/level from the appender.